app/controllers/draft.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const User =require('../models/user');
+const Draft=require('../models/draft');
+
+const response = require('../lib/responses');
+
+
+/**
+ * Adds the Draft to the table
+ * @param  {Express.Request}   	req  - the request object
+ * @param  {Express.Reponse}   	res  - the response object
+ * @param  {Function} 			next - pass to next handler
+ */
+
+function getDraft(req, res, next){
+    //check submitter_id
+	if(!req.body.submitter_id){
```

Comment:
This is my fault for not updating the story, but we're not bundling the user's id in with the request body. We're getting it from the JWT. The middleware decodes the user_id from the JWT, retrieves the matching user from our database, then attaches them to the request body.

Basically, you don't need this check.